tests/integration/views/test_questionnaire.py

```diff
@@ -538,6 +538,66 @@ def test_build_view_context_for_question(self):
         # Then
         self.assertEqual(question_view_context['question_titles']['single-title-question'], 'How are you feeling??')
 
+    def test_calculated_summary_answer_label(self):
```

Comment:
I would have expected a change to the functional test for this, not an integration test. This is a html change of what the user sees not the context that is passed. So essentially this test in `calculated_summary.spec.js`

Reply:
I tried, but the label field doesn't have an id (and thus no webdriver object).  So I thought an integration test was the only way to do it?
From what you said though, I'm going to assume there is.  Can you let me know what that is exactly?

Reply:
You're right there isn't but nothing stopping you adding one to display-answer-label.html

Reply:
Or the very catchy selector of
`#main > div.summary > div > div:nth-child(2) > div:nth-child(1) > div.grid__col.col-12\40 xs.col-6\40 m > div` but I wouldn't recommend using that.